src/Component/Rule/Rule.tsx

```diff
@@ -53,6 +54,7 @@ interface RuleDefaultProps {
   /** The data projection of example features */
   dataProjection: string;
   rendererType: 'SLD' | 'OpenLayers';
+  sldRendererProps: SLDRendererProps;
```

Comment:
Very minor (and probably not needed at the moment), but could you also provide an option to pass props for the `Renderer` (e.g. as `rendererProps`)? You decide.